core/domain/role_services.py

```diff
@@ -193,8 +193,7 @@
         ACTION_PUBLISH_ANY_ACTIVITY,
         ACTION_PUBLISH_OWNED_SKILL,
         ACTION_VISIT_ANY_QUESTION_EDITOR_PAGE,
-        ACTION_VISIT_ANY_TOPIC_EDITOR_PAGE,
```

Comment:
Updating rights so that voiceover admins can only access the voiceover admin page.